kyo-offheap/native/build.sbt

```diff
@@ -0,0 +1,15 @@
+   // kyo/kyo-offheap/native/build.sbt
+   name := "KyoOffheapNative"
```

Comment:
You could name the project `scala-native-panama` (just as we have `scala-native-java-logging` facade for Java's logging).

I think it would make sense to move this project on the same level with the others.